src/components/cc-zone/cc-zone.js

```diff
@@ -179,7 +179,6 @@ export class CcZone extends LitElement {
           background-color: var(--cc-zone-tag-bgcolor, var(--cc-color-bg-soft, #eeeeee));
           border: 1px solid var(--cc-zone-tag-bdcolor, transparent);
           border-radius: 0.25em;
-          border-radius: 0.25rem;
```

Comment:
Decided not to change it because to get back to a round 4px size, the `em` would get quite complex (more than 3 decimals).
As long as it's does not cause any visual issue, I decided to keep simple values.